Sources/Fullscreen/Explore/ExploreView.swift

```diff
@@ -222,4 +233,5 @@ extension ExploreView: RemoteImageViewDataSource {
 private enum Item: Equatable, Hashable {
     case regular(ExploreCollectionViewModel, ExploreCollectionCell.Kind)
     case tagCloud([TagCloudCellViewModel])
+    case brazeBanner(ExploreCollectionViewModel)
```

Comment:
Can `brazeBanner` have its own view model instead of reusing `ExploreCollectionViewModel`? I see the same was done for `tagCloud`

I know explore is built in a weird way around this view model, but hoping it's possible without too much work 🤞

Reply:
Good idea, it wasn't too much work either 👍